scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua

```diff
@@ -8,8 +8,10 @@ require("scripts/globals/mobs")
 local entity = {}
 
 entity.onMobInitialize = function(mob)
+    mob:setMod(xi.mod.UFASTCAST, 100)
```

Comment:
He doesn't have perma chainspell speed on, he casts very fast yes, but not instant. it is _very fast_ I'd estimate 75 to 90 eyeballing - if we could frame by frame a a few casts in a vid we can ballpark it from that vs a non-enhanced cast time.

Reply:
I reduced his Fastcast down to 75. Seems to match up what I see on videos of people fighting him.

Reply:
I have also added in a check to see whether or not Lumber Jack died or just despawned to set Weeping Willow's respawn correctly.

Reply:
> I have also added in a check to see whether or not Lumber Jack died or just despawned to set Weeping Willow's respawn correctly.

much appreciated, those of us that know this happens usually forget. Memory fuzzy, but I don't think its a static 30 min. But its acceptable till someone has concrete data.